src/components/safe-apps/AppCard/index.tsx

```diff
@@ -117,19 +127,13 @@ const CompactAppCard = ({ url, safeApp, onPin, pinned }: CompactSafeAppCardProps
 
 const AppCard = ({ safeApp, pinned, onPin, onDelete, variant = 'default' }: AppCardProps): ReactElement => {
   const router = useRouter()
-  const chainId = useChainId()
+  const currentChain = useCurrentChain()
 
-  const shareUrl = `${SAFE_REACT_URL}/share/safe-app?appUrl=${safeApp.url}&chainId=${chainId}`
+  const shareUrl = `${window.location.origin}${AppRoutes.share.safeApp}?appUrl=${safeApp.url}&chainId=${currentChain?.chainId}&chain=${currentChain?.shortName}`
```

Comment:
Do we need both `chainId` and `chain` as query params or is `chain` sufficient?

Reply:
I replaced it in favor of `chain`